### ISSUE TITLE
34978028: UITests are unable to find elements under transparent NSView.

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#18687]
#### Description

Area:
Xcode

Summary:
UITests are unable to find elements under transparent NSView.

Steps to Reproduce (see attached sample project):
1. Unzip attached "AccessibilityTest.zip".
2. Run testButtonPresenceByAccessibilityIdentifier test in AccessibilityTestUITests.swift

Expected Results:
Button could be found and clicked by UITests.

Actual Results:
Button cannot be clicked via UITests.
(Unable to find hit point for element Button, 0x60c000180ea0, {{357.0, 560.0}, {81.0, 32.0}}, identifier: 'ButtonWithAccessibility', title: 'Button')

Additional Info:
Worked in Xcode 8.x
The problem is only in UITests. For regular build run, button can be clicked. Also button can be clicked via Voice Over.

Version/Build:
Xcode Version 9.1 beta (9B46)

Configuration:
10.13.1 Beta (17B35a)

Sample Project:
AccessibilityTest.zip - goo.gl/jVV6nN

-
Product Version: 9.1 beta (9B46)
Created: 2017-10-17T08:14:06.921380
Originated: 2017-10-13T00:00:00
Open Radar Link: http://www.openradar.me/34978028
